src/services/balancer/subgraph/balancer-subgraph.client.ts

```diff
@@ -1,12 +1,13 @@
 import axios from 'axios';
+import { Container } from 'typedi';
 import { jsonToGraphQLQuery } from 'json-to-graphql-query';
-import { configService as _configService } from '@/services/config/config.service';
+import { ConfigService } from '@/services/config/config.service';
 
 export default class BalancerSubgraphClient {
   url: string;
 
-  constructor(private readonly configService = _configService) {
-    this.url = configService.network.subgraph;
+  constructor(private readonly configService = Container.get(ConfigService)) {
```

Comment:
Can this not be:
```suggestion
  constructor(private readonly configService = ConfigService) {
```
I thought this was partly the point of TypeDI, to clean up the dependency injection.

Reply:
To my understanding that would be possible if `BalancerSubgraphClient` was a `@Service` itself, something I felt was beyond the scope of this ticket as it has implications on how it can be used (not instantiated in a singleton).

From the docs (https://docs.typestack.community/typedi/02-basic-usage-guide) it mentions:

```
IMPORTANT NOTE:
Don't forget to annotate your classes with the @Service decorator! Both the ones being injected and those which requests the dependencies should be annotated.
```